### PR TITLE
Guess code subdirectory when importing repo

### PR DESCRIPTION
**Pull request by peteruhnak**
* When importing a local repo, guess the code subdirectory if there is .filetree file present.
  * This is a convenience fix to save some typing
* Fixed button label for import local repo

p.s. I am not sure to which branch I should direct the PR... dev-0.4 is apparently the working branch... so here?

### CODE REVIEW

File: Iceberg-UI.package/IceCreateRepositoryModel.class/instance/initializePresenter.st [hunk 1]
```diff
@@ -1,3 +1,7 @@
 initialization
 initializePresenter
-	createButton action: [ self createRepository ]
\ No newline at end of file
+	createButton action: [ self createRepository ].
+	localDirectoryLocation locationInput
+		whenTextChanged: [ :newText | 
+			self location isNotNil & subdirectory text isEmpty
+				ifTrue: [ subdirectory text: (self guessSubdirectoryIn: self location) ] ]
```
Comment (estebanlm) [inline Iceberg-UI.package/IceCreateRepositoryModel.class/instance/initializePresenter.st]: some remarks: 
- I do not like logic so complex in an event block, I'd prefer to extract a method there.
- this: `self location isNotNil & subdirectory text isEmpty`
is in general incorrect (unnecessary). It should be better like thos: 
`self location notNil and: [ subdirectory text isEmpty ]`

**Comment by peteruhnak** [comment]
Extracted (also I split the decision into two to avoid parenthesis and extra ifs)

Btw. I wanted to write test for this, but there are no tests in Iceberg, or am I missing something?

**Comment by estebanlm** [comment]
yes there are tests. 
If you install `#development` it will install everything to test...
I think some of them are broken right now... there is a problem with VM version on smalltalk-ci and our capability of test things in linux... but we need to work on restore it (just... I didn't have the time).

**Comment by peteruhnak** [comment]
I was fixing my branches, so this was automatically closed... I'll reopen it later with tests...

**Pull request closed without merging**